src/chapters/exceptions/exercises.rst

```diff
@@ -0,0 +1,29 @@
+Exercises
+=========
+
+To get started, fork and clone this `repo <https://github.com/LaunchCodeEducation/csharp-web-dev-lsn9exceptions>`_ from Github. 
+
+Divide By Zero!
+---------------
+
+Complete a function called ``Divide()`` in ``Program.cs``.
+``Divide()`` takes in two parameters: ``x`` and ``y``.
+
+Your function should return the result of ``x/y``.
+
+However, if ``y`` is zero you should throw an appropriate error message.
```

Comment:
change error message to exception. The exception will include a custom error message that they write.

Reply:
Done!